recipes/fusion-report/meta.yaml

```
@@ -9,43 +9,42 @@ source:
   sha256: ea6a3274c427760240a465052cbec549bddade35a37386d9c9dd08858fef6dfe
 
 build:
-  number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps --no-cache-dir -vvv
+  number: 1
+  noarch: generic
+  script: python3 setup.py install --single-version-externally-managed --record=record.txt
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps --no-cache-dir -vvv
```
This should be reversed